src/asset-management/AaveManager.sol

```diff
@@ -28,8 +28,10 @@ contract AaveManager is IAssetManager, Owned(msg.sender), ReentrancyGuard {
 
     /// @dev percentage of the pool's assets, above and below which
     /// the manager will divest the shortfall and invest the excess
-    uint256 public upperThreshold = 70;
-    uint256 public lowerThreshold = 30;
+    /// 1e18 == 100%
+    // TODO: can we use uint128 for each of these for gas savings?
```

Comment:
would like your opinion on this @OliverNChalk

Reply:
Yeah, this should save gas and I'm happy with changing

Reply:
okay never mind, changing it actually INCREASED gas so leaving it for now

Reply:
Okay, so your gas test does not conclude anything then. Happy to leave as uint256 as we can always iterate

Reply:
for record purposes, we have use the `gasleft()` method to show that it indeed decreases gas usage and thus we've implemented it